imagetagger/imagetagger/annotations/static/annotations/js/annotations.js

```diff
@@ -72,7 +71,7 @@ function calculateImageScale() {
   }
 
   function initTool() {
-    setTool();
+    setTool()
```

Comment:
Why remove the semicolon 🙃

Reply:
Ah, did it by mistake. I will get that fixed ASAP. Sorry!